video/system-test/analyze.test.js

```diff
@@ -24,50 +24,58 @@ const tools = require(`@google-cloud/nodejs-repo-tools`);
 const cmd = `node analyze.js`;
 const cwd = path.join(__dirname, `..`);
 
+const url = `gs://nodejs-docs-samples-video/quickstart.mp4`;
+const shortUrl = `gs://nodejs-docs-samples-video/quickstart_short.mp4`;
+const file = `resources/cat.mp4`;
+
 // analyze_faces
-test(`should analyze faces in a GCS file`, async (t) => {
-  const output = await tools.runAsync(`${cmd} faces gs://demomaker/larry_sergey_ice_bucket_short.mp4`, cwd);
+test.serial(`should analyze faces in a GCS file`, async (t) => {
+  const output = await tools.runAsync(`${cmd} faces ${url}`, cwd);
   t.regex(output, /Thumbnail size: \d+/);
   t.regex(output, /Start: \d+\.\d+s/);
   t.regex(output, /End: \d+\.\d+s/);
+  t.regex(output, /Time \d+\.\d+s: \(\d+, \d+\) - \(\d+, \d+\)/);
 });
 
 // analyze_labels_gcs (one scene)
-test(`should analyze labels in a GCS file with one scene`, async (t) => {
-  const output = await tools.runAsync(`${cmd} labels-gcs gs://demomaker/tomatoes.mp4`, cwd);
-  t.regex(output, /Label Tomato occurs at:/);
+test.serial(`should analyze labels in a GCS file with one scene`, async (t) => {
+  const output = await tools.runAsync(`${cmd} labels-gcs ${shortUrl}`, cwd);
+  t.regex(output, /Label shirt occurs at:/);
   t.regex(output, /Entire video/);
```

Comment:
We might no longer show "Entire video" in the output.  (see comments above about time offsets)